frontend/src/components/modules/LevelDots/LevelDots.tsx

```diff
@@ -1,7 +1,7 @@
-import { AdvancementLevelProps } from './AdvancementLevel.interface';
+import { LevelDotsProps } from './LevelDots.interface';
 import { MAX_LEVEL, MIN_LEVEL } from './constants';
 
-export const AdvancementLevel = ({ level }: AdvancementLevelProps) => {
+export const LevelDots = ({ level }: LevelDotsProps) => {
```

Comment:
we can replace it like we have in other components:
```suggestion
export const LevelDots: React.FC<LevelDotsProps> = ({ level }) => {
```